spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerProperties.java

```diff
@@ -289,18 +329,26 @@ public Integer getDebugPort() {
 		return debugPort;
 	}
 
-	public String getDebugSuspend() {
+	public DebugSuspendType getDebugSuspend() {
 		return debugSuspend;
 	}
 
-	public void setDebugSuspend(String debugSuspend) {
+	public void setDebugSuspend(DebugSuspendType debugSuspend) {
 		this.debugSuspend = debugSuspend;
 	}
 
 	public void setDebugPort(Integer debugPort) {
 		this.debugPort = debugPort;
 	}
```

Comment:
Can we log a warning here `'port' property is deprecated. Use 'address' instead`.

Reply:
done